howso/engine/trainee.py

```diff
@@ -3584,7 +3606,7 @@ def get_distances(
 
                 {
                     'case_indices': [ session-indices ],
-                    'distances': DataFrame( distances )
+                    distances=DataFrame( distances )
```

Comment:
Not sure why this line was changed, lets revert back

Reply:
Probably a regex find/replace artifact. Addressed by: 9d1dad2